### ISSUE TITLE
Tests

### ISSUE THREAD

**Issue by ArchBlood** [humhub/jwt-sso#4]
Would it be possible to set up some tests for this module, namely for implementing #3?

**Comment by luke-** [comment]
Yes, it would be very good to have tests here. Ideally also for the new configuration settings.
